### PR TITLE
WP-21444: Update splunk logs

### PR DESCRIPTION
**Pull request by dgomez5342**
# Description of change
https://varicent.atlassian.net/browse/WP-21444

- added logs to an array to print them as a single log to save space on splunk

### CODE REVIEW

**Review by JiawenHou — APPROVED** [review]

File: tap_s3_csv/__init__.py [hunk 1]
```diff
@@ -82,7 +83,7 @@ def do_sync(config, catalog, state):
             current_col_count = len(stream['schema']["properties"].items())
             total_col_count += current_col_count
             json_row_col = {"name": name, "stream_id":stream_name, "row": counter_value, "col": current_col_count}
-            LOGGER.info("EXPORTS tap-s3-csv individual_file_data_props: " + str(json_row_col))
+            grouped_logs.append("individual_file_data_props: " + str(json_row_col) + "\n")
```
Comment (JiawenHou) [inline tap_s3_csv/__init__.py:86]: not sure if export icm supports multiple streams in the catalog, if yes, may also want to include the stream name into the log?

File: tap_s3_csv/__init__.py [hunk 2]
```diff
@@ -94,7 +95,8 @@ def do_sync(config, catalog, state):
     
     # Exports logs for row and col count
     json_row_col = {"name": name, "row": total_row_count, "col": total_col_count}
-    LOGGER.info("EXPORTS tap-s3-csv data_props: " + str(json_row_col))
+    grouped_logs.insert(0,"EXPORTS tap-s3-csv data_props: " + str(json_row_col) + "\n")
+    LOGGER.info("|".join(grouped_logs))
```
Comment (JiawenHou) [inline tap_s3_csv/__init__.py:99]: guess we no longer need the `|` if we already append new row for each line of log?

Comment (JiawenHou) [reply tap_s3_csv/__init__.py:99]: one question for the `total_col_count`, say we have two sheets one with 10 columns, one with 20 columns, does the total_col_count just do 10+20?

Comment (dgomez5342) [reply tap_s3_csv/__init__.py:99]: Yes we would have that for quick checks. We would also have the individual ones if we want a more accurate look.

Comment (dgomez5342) [reply tap_s3_csv/__init__.py:99]: I can change it to use max columns if you think it would be better

Comment (JiawenHou) [reply tap_s3_csv/__init__.py:99]: that should be fine, just fear this `total_col_count` value could be miss leading.

Comment (dgomez5342) [reply tap_s3_csv/__init__.py:99]: Also need the | for script to easier separate the object that we want in it

**Review by JiawenHou — APPROVED** [review]

**Pull request merged**